_source/atoms/paragraph/Paragraph.scss

```diff
@@ -3,8 +3,12 @@
   font-family: $font__family--text;
   line-height: 1.556rem;
   padding-bottom: 1rem;
+}
+
+.paragraph--first {
```

Comment:
Is this the `:first-child` of a `<p>`?

Reply:
Nope, it's a prop to add some more space to the top of a paragraph. It's useful for a first paragraph.